index 96ff1e9f6a9..d67d709cd0a 100644
--- a/bfd/opncls.c
+++ b/bfd/opncls.c
@@ -1705,7 +1705,7 @@ bfd_create_gnu_debuglink_section (bfd *abfd, const char *filename)
     }
 
   /* Strip off any path components in filename.  */
-  filename = lbasename (filename);
+  //filename = lbasename (filename);
 
   sect = bfd_get_section_by_name (abfd, GNU_DEBUGLINK);
   if (sect)
@@ -1799,7 +1799,7 @@ bfd_fill_in_gnu_debuglink_section (bfd *abfd,
 
   /* Strip off any path components in filename,
      now that we no longer need them.  */
-  filename = lbasename (filename);
+  //filename = lbasename (filename);
 
   filelen = strlen (filename);
   debuglink_size = filelen + 1;
